mlos_core/mlos_core/optimizers/bayesian_optimizers/smac_optimizer.py

```diff
@@ -91,6 +103,22 @@ def __init__(self, *,  # pylint: disable=too-many-locals,too-many-arguments
         n_random_probability: float
             Probability of choosing to evaluate a random configuration during optimization.
             Defaults to `0.1`. Setting this to a higher value favors exploration over exploitation.
+
+        facade: AbstractFacade
+            sets the facade to use for SMAC
```

Comment:
```suggestion
            Sets the facade to use for SMAC.
```

Maybe also provide a small hint on what a "facade" is, or just link to the appropriate docs in SMAC?